### ISSUE TITLE
🚀 Keybinding to jump between files in navigate mode

### ISSUE THREAD

**Issue by tjkirch** [dandavison/delta#680]
Would it be possible to add additional keybindings that jump to the next or previous file in a diff?  Some files have many diff hunks (such as generated files, like Cargo.lock in Rust) and it'd be nice to be able to skip down to the next file.

I see that the keybindings for navigating hunks insert a search pattern in less's history, and I don't see a way for less to have multiple patterns or keybindings, so I couldn't personally see a way without switching pager.

**Comment by dandavison** [comment]
Hi @tjkirch! I think you're right -- it looks like we need to expose a bit more control over the details of the `navigate` regexp. E.g. unmerged commit dfc149c gives us distinct labels for file and hunk boundaries, but we also need to give users the ability to alter the regex itself that delta seeds less with, in order to specify stopping on file boundaries _only_.

**Comment by lepotic** [comment]
@dandavison I think it's also worth thinking about the order of the labels in the regex.
If we have
```
^(commit|added:|removed:|renamed:|Δ|δ)
```
(the hunk label is last), it will be easy to switch to jumping by files by pressing /, Up, and removing the last couple of chars and pressing Enter.

**Comment by dandavison** [comment]
Nice @lepotic. Thanks very much for that suggestion. I've made your change and opened PR https://github.com/dandavison/delta/pull/684. I'm thinking that gives us a pretty workable solution; @tjkirch @lepotic what do you think? https://github.com/dandavison/delta/pull/684 PR description copied below:

----------------------------------------------------------

Fixes #680 cc @tjkirch @lepotic

This changes the labels that delta adds when `navigate = true`: we now have an upper-case delta on file boundaries, and a lower-case delta on hunk boundaries:

<table><tr><td><img width=300px src="https://user-images.githubusercontent.com/52205/129232414-d0d9f892-df5d-4ff9-80ce-086a71724353.png" alt="image" /></td></tr></table>

As suggested by @lepotic https://github.com/dandavison/delta/issues/680#issuecomment-897489803, this allows us to switch to navigating by file with `/ <up> <left> <backspace> <backspace> <enter>`.

In other words, using `/ <up>` to reveal the current search regex:
```
/^(commit|added:|removed:|renamed:|Δ|δ)
```
and then editing this to remove the lower-case delta:

```
/^(commit|added:|removed:|renamed:|Δ)
```

and finally resuming search with `<enter>`.

The labels can be modified. E.g.

```
[delta]
    file-modified-label = @
    hunk-label = *
```

(and it is ok to use something like `*` which has special meaning in regular expressions).
